.github/workflows/sync-openapi-spec.yml

```
@@ -49,7 +49,7 @@ jobs:
         continue-on-error: true
         uses: actions/checkout@v6
         with:
-          repository: ansible-automation-platform/aap-openapi-specs
+          repository:  ${{ secrets.OPENAPI_SPEC_REPO }}
```

actionlint (1.7.12)

[error] property "openapi_spec_repo" is not defined in object type {actions_runner_debug: string; actions_step_debug: string; github_token: string; openapi_spec_sync_token: string}

(expression)